tests/migration/online_migration/pre_migration.pm

```diff
@@ -12,12 +12,14 @@ use strict;
 use warnings;
 use testapi;
 use Utils::Architectures;
+use Utils::Backends;
 use utils;
 use migration;
 use version_utils;
 use x11utils 'turn_off_gnome_show_banner';
 
 sub check_or_install_packages {
+    script_run('modprobe nvram') if is_pvm_hmc;
```

Comment:
i suggest to use `assert_script_run` here. Such, it is the recommended way to assert that an execution run correctly.